tools/SourceKit/lib/SwiftLang/CodeCompletionOrganizer.h

```diff
@@ -46,6 +46,8 @@ struct Options {
   bool annotatedDescription = false;
   bool includeObjectLiterals = true;
   bool addCallWithNoDefaultArgs = true;
+  bool verifyUSRToDecl = false;
+  bool includeFullDocumentation = false;
```

Comment:
I don't see anything is actually using this flag now. I believe this is a leftover from the previous implementation. Could you remove this?

Reply:
It's used for testing full documentation retrieval using `sourcekitd-test`.

Reply:
Oh I see, sorry I misunderstood. That being said, I don't think we should have this option since we don't want anything would use it as SourceKit API. For testing the full documentation rendering, I believe `swift-ide-test` is enough. I'd prefer not to have sourcekitd-test testing the `doc.full_as_xml`. @hamishknight @bnbarham wdyt?

Reply:
Yeah, I think swift-ide-test + sourcekit-lsp testing is enough here. The sourcekitd-test as is doesn't really test anything that swift-ide-test isn't already.